engines/download_only.py

```diff
@@ -81,6 +85,7 @@ def insert_data_from_url(self, url):
             self.download_file(url, filename)
 
     def find_file(self, filename):
+        """Checks for the given file"""
```

Comment:
`"""Checks for the given file"""` -> `"""Checks for the given file and adds it to the list of all files"""`